packages/hop-node/src/config/config.ts

```diff
@@ -301,6 +303,13 @@ export const setNetworkRpcUrl = (network: string, rpcUrl: string) => {
   }
 }
 
+export const setNetworkRedundantRpcUrls = (network: string, redundantRpcUrs: string[]) => {
```

Comment:
Typo in var `redundantRpcUrs`

```suggestion
export const setNetworkRedundantRpcUrls = (network: string, redundantRpcUrls: string[]) => {
```